Azure/communication-ui-library-ios
# [Bugfix] Fix placeholder text view blocks TextEditor

## Purpose

* This is to fix when users type on the left half of the text input view on a thread, the keyboard will not show up.
* The reason being that, the text view for placeholder rendered on top of the TextEditor view. So it blocks the interaction.

## Does this introduce a breaking change?



## Pull Request Type
What kind of change does this Pull Request introduce?



## Pull request checklist

This PR has considered:

## How to Test


* Open a chat thread
* Click on the bottom text input view
* Keyboard should show up


## What to Check

| Before                                       | After                                      |
|----------------------------------------------|--------------------------------------------|
| Screenshot or description before this change | Screenshot or description with this change |

<img src="https://user-images.githubusercontent.com/107075081/207746680-35c5786f-6322-4823-a295-cb6a44badfb1.PNG" width=200 height=400> | <img src="https://user-images.githubusercontent.com/107075081/207747191-a290f34e-12ff-4549-aece-2e209cfdd2ed.PNG" width=200 height=400>
